src/main/java/eu/dzhw/fdz/metadatamanagement/analysispackagemanagement/domain/AnalysisPackage.java

```diff
@@ -209,6 +209,11 @@ public class AnalysisPackage extends AbstractShadowableRdcDomainObject
   @NotNull(message = "analysis-package-management.error.analysis-package.tags.not-null")
   private Tags tags;
 
+  /**
+   * The DOI of the analysis package.
+   */
+  private String doi;
```

Comment:
@ewolf47 as per our discussion: we should keep this in the document child class to avoid triggering unwanted shadow copies.